From 400a976aba7dccbd07b25057029b759cf6430a65 Mon Sep 17 00:00:00 2001
From: Mitch Luban <mluban@nvidia.com>
Date: Tue, 29 Jan 2013 18:32:39 -0800
Subject: [PATCH] power: Add POWER_HINT_APP_LAUNCH to power hints

Need POWER_HINT_APP_LAUNCH to boost on app launch.

Bug 1155935
Bug 1206410

Change-Id: I35f260764151edad27df645384756f9c4527a1e5
Reviewed-on: http://git-master/r/195366
Tested-by: Mitch Luban <mluban@nvidia.com>
Reviewed-by: Prajakta Gudadhe <pgudadhe@nvidia.com>
---
 include/hardware/power.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index 193ba68..441b726 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -42,7 +42,8 @@ typedef enum {
     POWER_HINT_VSYNC = 0x00000001,
     POWER_HINT_INTERACTION = 0x00000002,
     POWER_HINT_APP_PROFILE = 0x00000003,
-    POWER_HINT_COUNT,
+    POWER_HINT_APP_LAUNCH = 0x00000004,
+    POWER_HINT_COUNT = 0x00000005,
 } power_hint_t;
 
 /**
-- 
1.8.1.5

